perfkitbenchmarker/vm_util.py

```diff
@@ -246,10 +246,10 @@ def WrappedFunction(*args, **kwargs):
           sleep_time = poll_interval * fuzz_multiplier
           if ((time.time() + sleep_time) >= deadline or
               (max_retries >= 0 and tries > max_retries)):
-            raise e
+            raise
```

Comment:
Sorry, I dont fully understand the implications of this change. Ideally we would want to raise the same root cause exception without masking it, and make it clear what we are doing. Does removing the exact exception have any effect ?